src/scripts/main.js

```diff
@@ -1 +1,15 @@
 'use strict';
+
+const largeIMG = document.getElementById('largeImg');
+
+const smallIMG = document.getElementById('thumbs');
+
+smallIMG.addEventListener('click', (evnt) => {
+  evnt.preventDefault();
```

Comment:
Consider moving `evnt.preventDefault();` inside the `if` block to ensure it only prevents the default action for anchor elements. This will avoid unnecessary prevention of default actions for non-anchor elements.